crates/sargon/src/profile/mfa/mfa_factor_sources/security_questions_factor_source/security_questions_factor_source.rs

```diff
@@ -279,6 +279,10 @@ impl BaseIsFactorSource
     fn set_common_properties(&mut self, updated: FactorSourceCommon) {
         self.common = updated
     }
+
+    fn name(&self) -> String {
+        "NOT IMPLEMENTED".to_owned()
```

Comment:
Maybe we should use this instead:

```rust
    fn name(&self) -> String {
        let ids = self
            .sealed_mnemonic
            .security_questions
            .items()
            .into_iter()
            .map(|q| q.id())
            .map(|id| format!("#{:?}", id))
            .join(", ");
        format!("Questions: {}", ids)
    }
```

It ensures if we ever were to forget about this, we at least have something which is somewhat decent.

Reply:
Why not have a hint the same way we do for other factor sources?

Reply:
We might, but let’s postpone this until we implement security questions and use this default name for now.